cli/cmd/sync.go

```diff
@@ -276,6 +282,10 @@ func sync(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return enrichClientError(managedplugin.Clients{}, []bool{destination.RegistryInferred()}, err)
 		}
+		latestVersion, err := destPluginClient.FindLatestPluginVersion(ctx, managedplugin.PluginDestination)
+		if err == nil && destination.Version != "" && latestVersion != "" && semver.IsLess(destination.Version, latestVersion) {
+			log.Warn().Str("source", destination.Name).Msgf("Your destination plugin is out of date, you are using version %s, and the latest version is %s. Consider upgrading.", destination.Version, latestVersion)
```

Comment:
Including a link to the hub would look like:
```suggestion
			log.Warn().Str("source", destination.Name).Str("url", "https://hub.cloudquery.io/plugins/destination/" + destination.Path).Msgf("Your destination plugin is out of date, you are using version %s, and the latest version is %s. Consider upgrading.", destination.Version, latestVersion)
```

Reply:
Fixed